Source/WebCore/en.lproj/Localizable.strings

```diff
@@ -1911,4 +1920,3 @@
 
 /* Option in segmented control for choosing list type in text editing */
 "•" = "•";
-
```

Comment:
Please undo this change.

Reply:
Not even sure how i did this lol, removed